packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -235,12 +265,34 @@ SelfUtils.isAdmittedGuest = (oldSelf, changedSelf) => {
   return SelfUtils.isLocusGuestUnadmitted(oldSelf) && SelfUtils.isLocusGuestAdmitted(changedSelf);
 };
 
-SelfUtils.mutedByOthers = (oldSelf = {}, changedSelf) => {
+SelfUtils.mutedByOthersChanged = (oldSelf = {}, changedSelf) => {
   if (!changedSelf) {
     throw new ParameterError('New self must be defined to determine if self was muted by others.');
   }
 
-  return changedSelf.muted && (oldSelf.lastModified !== changedSelf.lastModified || oldSelf.modifiedBy !== changedSelf.modifiedBy);
+  if (!oldSelf && !changedSelf.remoteMuted) {
+    // we don't want to be sending the 'meeting:self:unmutedByOthers' notification on meeting join
+    return false;
+  }
+
+  return (oldSelf.remoteMuted !== changedSelf.remoteMuted) || (changedSelf.remoteMuted && (oldSelf.unmuteAllowed !== changedSelf.unmuteAllowed));
```

Comment:
because we have unmuteAllowed field in the 'meeting:self:mutedByOthers' event, so when either remoteMuted or unmuteAllowed changes, we have to send a new 'meeting:self:mutedByOthers' event